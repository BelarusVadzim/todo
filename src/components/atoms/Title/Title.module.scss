@import 'src/variables.scss';

.title {
  border: none;
  color: rgb(255, 255, 255);
  cursor: default;
  font-family: Arial, Helvetica, sans-serif;
  font-size: 30px;
  font-weight: bold;
  letter-spacing: 10px;
}

