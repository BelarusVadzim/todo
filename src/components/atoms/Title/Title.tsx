import { PropsWithChildren } from "react";
import style from "./Title.module.scss";

type TitleProps = PropsWithChildren;

const Title: React.FC<TitleProps> = ({ children }) => {
  return <span className={style.title}>{children}</span>;
};

export default Title;
