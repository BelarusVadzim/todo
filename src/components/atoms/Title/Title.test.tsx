import render  from 'react-test-renderer';
import Title from './Title';

jest.mock('./Title.module.scss', () => ({ title: 'titleStyle' }));

describe('<Title />', () => {
  it('should render properly', () => {
    const component = render
      .create(<Title>Title text</Title>);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
