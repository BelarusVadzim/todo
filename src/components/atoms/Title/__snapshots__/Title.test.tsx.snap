// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render properly 1`] = `
<span
  className="titleStyle"
>
  Title text
</span>
`;
